sky/serve/autoscalers.py

```diff
@@ -361,6 +394,8 @@ def evaluate_scaling(
         # Case 1. Once there is min_replicas number of
         # ready new replicas, we will direct all traffic to them,
         # we can scale down all old replicas.
+        # Or, if rolling update is in progress, we scale down old replicas
+        # based on the number of ready new replicas.
```

Comment:
```suggestion
        # Case 1. For blue and green update, once there is min_replicas number of
        # ready new replicas, we will direct all traffic to them,
        # we can scale down all old replicas.
        # for rolling update, we scale down old replicas
        # based on the number of ready new replicas.
```
i.e. explicitly saying only in `blue and green` we compare to `min_replicas`

Reply:
Good point! Updated.